src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/ActionBar.xaml.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.ObjectModel;
+using CommunityToolkit.Mvvm.ComponentModel;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+
+namespace Microsoft.CmdPal.UI.Controls;
+
+[ObservableObject]
+public sealed partial class ActionBar : UserControl
+{
+    [ObservableProperty]
+    private string _actionName = string.Empty;
+
+    [ObservableProperty]
+    private bool _moreCommandsAvailable = false;
```

Comment:
It could be good to start splitting these out now as we go into the ViewModels in the VM project (like the ListPage/ListViewModel)?

As we don't want to use the `[ObservableObject]` attribute on UI components. At least then we have some of the separation setup and can figure out how we hook the data into it after once we have the extensions loaded?